lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -211,6 +212,17 @@ def _get_user_label(self, user_id):
             None
         )
 
+    def _get_user_label_from_username(self, username):
+        """
+        Returns role label of user from username
+        Possible Role Labels: Staff, Community TA or None
+        """
+        try:
+            user = User.objects.get(username=username)
+            return self._get_user_label(user.id)
```

Comment:
good to add a user is null check here

Reply:
if user does not exist, it will raise ObjectDoesNotExist Exception, which returns None